tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -656,4 +656,10 @@ test_i2c_bh1750: bh1750@65 {
 test_i2c_akm09918c: akm09918c@66 {
 	compatible = "asahi-kasei,akm09918c";
 	reg = <0x66>;
+
```

Comment:
You deleted the node closure. Missing: `};`

Reply:
@erwango oh, thanks, this was where the conflict was with main, and I must have accidentally deleted it.

Reply:
@erwango I fixed this mistake. Would you mind approving? We have two approvals now, but the requested change is holding things up. Thanks for taking a look and for pointing to the exact reason the test was failing!